### PR TITLE
fix links

### PR DESCRIPTION
**Pull request by katrinleinweber**
might help with #120

### CODE REVIEW

File: languages/ccpp.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Any code editor will do. Try to avoid relying on IDEs too much; by helping your
 * Vim, recommended plugins: 
   + [NERDTree](https://github.com/scrooloose/nerdtree) file explorer.
   + [editorconfig](https://github.com/editorconfig/editorconfig-vim)
-  + [stl.vim](http://www.vim.org/scripts/script.php?script_id=4293) adds STL to syntax highlighting 
+  + [stl.vim](https://www.vim.org/scripts/script.php?script_id=4293) adds STL to syntax highlighting
```
Comment (nielsdrost) [inline languages/ccpp.md]: I think this link should be https://vim.sourceforge.io/scripts/script.php?script_id=4293

**Comment by nielsdrost** [comment]
Thanks! Still one broken link, shall I merge like this, or would you be willing to fix the last link too?

This Travis build really helps to keep the amount of broken links down, but the sheer number of links makes it hard to keep up. Appreciate the help :-)

**Comment by katrinleinweber** [comment]
https://vim.sourceforge.io/scripts/script.php?script_id=4293 is the link I added, after testing it. They have http://vim.sourceforge.io/scripts/script.php?script_id=4293 also, but that was "broken" before. I presume Travis is using a user-agent that they classify as a bot, and reject the connection.

So, IMHO this can be merged, unless the links are broken for you as well.

**Comment by nielsdrost** [comment]
Ok, lets merge like this. Definitely improves matters :-)

**Comment by nielsdrost** [comment]
Thanks again, updated some more links (#123) to finally get back to 0 dead links. :-)

**Comment by katrinleinweber** [comment]
🎉 You're welcome :-)

**Pull request merged**